ocient/README.md

```diff
@@ -0,0 +1,66 @@
+# Agent Check: Ocient
+
+## Overview
+
+[Ocient][1] Hyperscale Data Warehouse is a data analytics software solutions company that enables all-the-time, compute-intensive analysis of large, complex datasets while optimizing for performance, cost, and energy efficiency.
+
+With industry-standard interfaces like SQL and JDBC, Ocient makes it easy for organizations to interact with data within its platform.
+The Ocient integration enables you to collect metrics related to query performance, disk usage, database tables, and more.
+
```

Comment:
Could you add 1-2 sentences about what value this brings to our mutual customers?